doc/input-output-reference/src/overview/group-radiative-convective-units.tex

```diff
@@ -1309,12 +1309,33 @@ \subsubsection{Inputs}\label{inputs-6-019}
 \end{itemize}
 
-\emph{Operative temperature} for radiant system controls is the average of Mean Air Temperature and Mean Radiant Temperature.  The \emph{SurfaceFaceTemperature} option allows the user to control the radiant system using the inside face surface temperature of the radiant system (the inside surface temperature).  The \emph{SurfaceInteriorTemperature} option will allow the user to control the radiant system using a surface temperature that is calculated inside the radiant system.  This point will be defined by the  \hyperref[constructioninternalsource]{Construction:InternalSource} description for the system.  In that input, the user has the option to calculate a temperature at a particular point in the construction.  The radiant system will then use this information for controlling the slab not just producing temperatures for outputting at that point.  Users should consult the input for Field: Temperature Calculation Requested After Layer Number for more information.  Note that for the SurfaceFaceTemperature and SurfaceInteriorTemperature the surface being used for control is the surface listed above in the field for Surface Name.  If the user enters a group of surfaces for that input, the \emph{first} surface in the radiant group is the surface for control purposes.
+\emph{Operative temperature} for radiant system controls is the average of Mean Air Temperature and Mean Radiant Temperature.  The \emph{SurfaceFaceTemperature} option allows the user to control the radiant system using the inside face surface temperature of the radiant system (the inside surface temperature).  The \emph{SurfaceInteriorTemperature} option will allow the user to control the radiant system using a surface temperature that is calculated inside the radiant system.  This point will be defined by the  \hyperref[constructioninternalsource]{Construction:InternalSource} description for the system.  In that input, the user has the option to calculate a temperature at a particular point in the construction.  The radiant system will then use this information for controlling the slab not just producing temperatures for outputting at that point.  Users should consult the input for Field: Temperature Calculation Requested After Layer Number for more information.  Note that for the SurfaceFaceTemperature and SurfaceInteriorTemperature the surface being used for control is the surface listed above in the field for Surface Name.  If the user enters a group of surfaces for that input, the \emph{first} surface in the radiant group is the surface for control purposes.  The \emph{RunningMeanOutdoorAirTemperature} option will allow the user to control the inlet water temperature to the system as a function of the running mean outdoor air temperature.  The next field (\hyperref[field-running-mean-outdoor-air-temperature-weighting-factor]{Running Mean Outdoor Air Temperature Weighting Factor}) will define how this temperature is actually calculated within EnergyPlus.
```

Comment:
Given the long list of control types, this would be more readable using the `{itemize}` style (same as Condensation Control Type). 
And this should make it clear that it's averaging outdoor *dry bulb* temperature. Same for the next field.

Reply:
Will fix this--should show up in the next commit.